cabal-install/cabal-install.cabal

```diff
@@ -205,78 +185,21 @@ executable cabal
-    other-modules:
-        Distribution.Solver.Compat.Prelude
-        Distribution.Solver.Modular
-        Distribution.Solver.Modular.Assignment
-        Distribution.Solver.Modular.Builder
-        Distribution.Solver.Modular.Configured
-        Distribution.Solver.Modular.ConfiguredConversion
-        Distribution.Solver.Modular.ConflictSet
-        Distribution.Solver.Modular.Cycles
-        Distribution.Solver.Modular.Dependency
-        Distribution.Solver.Modular.Explore
-        Distribution.Solver.Modular.Flag
-        Distribution.Solver.Modular.Index
-        Distribution.Solver.Modular.IndexConversion
-        Distribution.Solver.Modular.LabeledGraph
-        Distribution.Solver.Modular.Linking
-        Distribution.Solver.Modular.Log
-        Distribution.Solver.Modular.Message
-        Distribution.Solver.Modular.PSQ
-        Distribution.Solver.Modular.Package
-        Distribution.Solver.Modular.Preference
-        Distribution.Solver.Modular.RetryLog
-        Distribution.Solver.Modular.Solver
-        Distribution.Solver.Modular.Tree
-        Distribution.Solver.Modular.Validate
-        Distribution.Solver.Modular.Var
-        Distribution.Solver.Modular.Version
-        Distribution.Solver.Modular.WeightedPSQ
-        Distribution.Solver.Types.ComponentDeps
-        Distribution.Solver.Types.ConstraintSource
-        Distribution.Solver.Types.DependencyResolver
-        Distribution.Solver.Types.Flag
-        Distribution.Solver.Types.InstSolverPackage
-        Distribution.Solver.Types.InstalledPreference
-        Distribution.Solver.Types.LabeledPackageConstraint
-        Distribution.Solver.Types.OptionalStanza
-        Distribution.Solver.Types.PackageConstraint
-        Distribution.Solver.Types.PackageFixedDeps
-        Distribution.Solver.Types.PackageIndex
-        Distribution.Solver.Types.PackagePath
-        Distribution.Solver.Types.PackagePreferences
-        Distribution.Solver.Types.PkgConfigDb
-        Distribution.Solver.Types.Progress
-        Distribution.Solver.Types.ResolverPackage
-        Distribution.Solver.Types.Settings
-        Distribution.Solver.Types.SolverId
-        Distribution.Solver.Types.SolverPackage
-        Distribution.Solver.Types.SourcePackage
-        Distribution.Solver.Types.Variable
-
-    other-modules:
-        Paths_cabal_install
-
     build-depends:
         async      >= 2.0      && < 2.3,
         array      >= 0.4      && < 0.6,
-        base       >= 4.8      && < 4.15,
         base16-bytestring >= 0.1.1 && < 1.1.0.0,
         binary     >= 0.7.3    && < 0.9,
         bytestring >= 0.10.6.0 && < 0.12,
         Cabal      == 3.5.*,
+        cabal-install-solver,
         containers >= 0.5.6.2  && < 0.7,
         cryptohash-sha256 >= 0.11 && < 0.12,
         deepseq    >= 1.4.1.1  && < 1.5,
         directory  >= 1.2.2.0  && < 1.4,
         echo       >= 0.1.3    && < 0.2,
         edit-distance >= 0.2.2 && < 0.3,
+        exceptions,
```

Comment:
where is this needed? I don't see any change that uses it. It also needs bounds

Reply:
Artifact from the Cabal Init work. Nice catch - i'll remove.